CHANGELOG.md

```diff
@@ -3,10 +3,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unreleased
+## 2.0.5 - 2017-12-14
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374)
```

Comment:
Would be useful to link to the new doc for this, https://github.com/oracle/terraform-provider-oci/blob/master/docs/Managing%20Default%20Resources.md